chameleon-server/src/main/java/com/chameleonvision/network/LinuxNetworking.java

```diff
@@ -1,73 +1,102 @@
 package com.chameleonvision.network;
 
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
 import java.io.IOException;
+import java.net.InetAddress;
 import java.net.SocketException;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 public class LinuxNetworking extends SysNetworking {
+    private static final String PATH = "/etc/dhcpcd.conf";
 
-	@Override
-	public boolean setDHCP() {
-		String[] clearArgs = { "addr", "flush", "dev", networkInterface.name };
-		try {
-			int clearRetCode = shell.execute("ip", clearArgs);
-			int dhcpRetCode = shell.execute("dhclient", networkInterface.name);
-			return clearRetCode == 0 && dhcpRetCode == 0;
-		} catch (IOException e) {
-			e.printStackTrace();
-			return false;
-		}
-	}
-
-	@Override
-	public boolean setHostname(String newHostname) {
-		String[] setHostnameArgs = { "set-hostname", newHostname };
-		try {
-			var setHostnameRetCode = shell.execute("hostnamectl", setHostnameArgs);
-			return setHostnameRetCode == 0;
-		} catch(Exception e)  {
-			e.printStackTrace();
-			return false;
-		}
-	}
+    @Override
+    public boolean setDHCP() {
+        File dhcpConf = new File(PATH);
+        if (dhcpConf.exists()) {
+            try {
+                List<String> lines = FileUtils.readLines(dhcpConf, StandardCharsets.UTF_8);
+                for (int i = 0; i < lines.size(); i++) {
+                    String line = lines.get(i);
+                    if (line.contains("interface " + networkInterface.name)) {
```

Comment:
Try checking using `startsWith(String)` to be sure we don't delete commented lines